src/Promact.TestCaseManagement.DomainModel/Models/TestCaseBase.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Promact.TestCaseManagement.DomainModel.Models
+{
+    public abstract class TestCaseBase : TestCaseManagementBase
+    {
+        public string TestCaseUniqueId { get; set; }
+
+        public string Description { get; set; }
+
+        public int? DefectId { get; set; }
+
+        public bool Priority { get; set; }
+
+        public string CreatedUserId { get; set; }
```

Comment:
Yes, CreatedUserId is a foreign key to CreatedUser and it will map without ForeignKey attribute.